pkg/agent/cniserver/server_windows.go

```diff
@@ -98,3 +99,15 @@ func getInfraContainer(containerID, netNS string) string {
 func (c *CNIConfig) getInfraContainer() string {
 	return getInfraContainer(c.ContainerId, c.Netns)
 }
+
```

Comment:
```suggestion
| Pod Type                         | Spec.HostNetwork | windowsOptions.hostProcess | Kubelet CMDADD  | OVS interface needed? | Reconcile needed?          |
| 1. Normal Pod (non-HostNetwork)  | false            | false / not set            | Yes             | Yes                   | Yes                        |
| 2. Windows HostNetwork Pod       | true             | false / not set            | Yes             | Yes                   | Yes                        |
| 3. Windows HostProcess Pod       | (any)            | true                       | No              | No                    | No (no real network config) | 
``` @wenyingd Can we add a table to the code comments to explain the different behaviors of Windows Pods for developers?

Reply:
updated.